src/components/input-field/input-field.tsx

```diff
@@ -479,4 +480,9 @@ export class InputField {
             this.action.emit();
         }
     }
+    private handleWheel(event) {
```

Comment:
We should add a type to the event
```suggestion
    private handleWheel(event: MouseEvent) {
```